packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -109,7 +114,30 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
               {originFromZero && (
                 <span className={css(classNames.zeroTick)} style={this._getStyleUsingOffsetPercent(vertical, zeroOffsetPercent)} />
               )}
-              <span ref={this._thumb} className={classNames.thumb} style={this._getStyleUsingOffsetPercent(vertical, thumbOffsetPercent)} />
+              {typeof marks === 'boolean' && marks && this._addTickmarks(css(classNames.regularTick))}
```

Comment:
Why do you need the boolean type check? If it’s `false` or `undefined` it will not render anything.